web/src/server/api/definitions/sessionsView.ts

```diff
@@ -3,38 +3,75 @@ import { type ColumnDefinition } from "@/src/server/api/interfaces/tableDefiniti
 export const sessionsViewCols: ColumnDefinition[] = [
   { name: "⭐️", id: "bookmarked", type: "boolean", internal: "s.bookmarked" },
   {
-    name: "id",
+    name: "Id",
     id: "id",
     type: "string",
     internal: 's."id"',
   },
   {
-    name: "userId",
+    name: "User Ids",
+    id: "userIds",
     type: "string",
     internal: "array_to_string(t.\"userIds\", ', ')",
```

Comment:
What do you think about re-using the tags filter here? I think having a drop down with userids would be most helpful.